Support for installing RHDH using Operator (#62)
redhat-developer/red-hat-developers-documentation-rhdh: getting-started/con-configuring-dynamic-plugins-operator.adoc

[id='con-configuring-dynamic-plugins-operator_{context}']

> = Configuring dynamic plugins with the Operator

Reviewer: I noticed that the Admin guide had a section dedicated to dynamic plugins, but it is currently Helm-oriented: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-62/admin/#rhdh-installing-dynamic-plugins
Would it make sense to move this under this section (maybe a separate "Dynamic Plugin installation using the Operator" section)?

> You can store the configuration for dynamic plugins in a ConfigMap object, which can then be referenced into the Custom Resource.

Reviewer 2: "referenced into" => "referenced by"

> You can store the configuration for dynamic plugins in a ConfigMap object, which can then be referenced into the Custom Resource.

Reviewer 2: ```suggestion
You can store the configuration for dynamic plugins in a ConfigMap object, which can then be referenced by the Custom Resource.
```

== Example using the GitHub dynamic plugin
In OpenShift, you can use the following content as a base template to create a ConfigMap named `dynamic-plugins-rhdh`:

[source,yaml]
----
kind: ConfigMap
apiVersion: v1
metadata:
  name: dynamic-plugins-rhdh
data:
  dynamic-plugins.yaml: |
    includes:
      - dynamic-plugins.default.yaml
    plugins:
      - package: './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic'
        disabled: false
        pluginConfig: {}
----

[NOTE]

> If the `pluginConfig` field references environment variables, they will need to be provided by including the variables in your `secrets-rhdh` secret.

Reviewer 2: "they will need to be provided by including the variables in your `secrets-rhdh` secret" => "they shall be defined in your `secrets-rhdh` secret".

> If the `pluginConfig` field references environment variables, they will need to be provided by including the variables in your `secrets-rhdh` secret.

Reviewer 2: ```suggestion
If the `pluginConfig` field references environment variables, they shall be defined in your `secrets-rhdh` secret.
```

Author: ```suggestion
If the `pluginConfig` field references environment variables, you must define the variables in your `secrets-rhdh` secret.
```
